src/AppShed/Extensions/SpreadsheetBundle/Resources/config/services.xml

```diff
@@ -52,8 +52,34 @@
                 <argument>false</argument>
             </call>
         </service>
+        <service id="app_shed_extensions_spreadsheet.geo_client"
+                 class="GuzzleHttp\Client"
+                 factory-method="get"
+                 factory-class="AppShed\Extensions\SpreadsheetBundle\DependencyInjection\GuzzleClientFactory">
+            <argument type="collection">
+                <argument key="base_url" type="collection">
+                    <argument>http://maps.google.com/maps/api/{action}</argument>
```

Comment:
At least 1 argument is required in <argument type="collection"> by GuzzleHttp\Client